docs/en/ingest-management/fleet-agent-proxy-support.asciidoc

```diff
@@ -272,6 +272,7 @@ fleet:
   agent:
     id: ""
 ----
+NOTE: Since version 8.3 of the Elastic Agent, after being read, the `fleet.yml` file will be encrypted and renamed to `fleet.enc`.
```

Comment:
```suggestion

NOTE: When {agent} runs, the `fleet.yml` file gets encrypted and renamed to `fleet.enc`.
```

I don't know if this is accurate ^^. I'm trying to simplify the statement because it's not clear what we mean by "after being read." I think you mean when Elastic Agent runs for the first time, it reads the file, then the file gets encrypted. 

I don't think we need to mention the version number; we can just backport this PR to 8.3.

Reply:
Looks good to me. :)